src/bots/ama/subCommands/addEvent.js

```diff
@@ -1,12 +1,18 @@
 import Event from '../models/Event';
 import client from '../../../client';
+import { isMod } from '../../../utils/perms';
 import { getFormattedEvent, getStrings } from '../constants';
 
 /**
  * Handles adding an event to Event schema and sends message with new event
  * @param {Array.<string>} args - rest of command arguments
  */
 const handler = async (args) => {
+  if (!isMod(client.message.member)) {
```

Comment:
is the `getMemberFromMessage` function overkill? maybe... let's decide whether or not we should keep it. if we do it'd be like:
```suggestion
  if (!isMod(getMemberFromMessage())) {
```

Reply:
but totally fine with not keeping it seeing as we can access client.message from anywhere the same way

Reply:
I missed that :( will change it